core/src/main/java/org/geysermc/geyser/command/defaults/ConnectionTestCommand.java

```diff
@@ -55,31 +56,56 @@ public void execute(@Nullable GeyserSession session, GeyserCommandSource sender,
         }
 
         if (args.length == 0) {
-            sender.sendMessage("Provide the Bedrock server IP you are trying to connect with. Example: `test.geysermc.org:19132`");
+            sender.sendMessage("Provide the Bedrock server IP and port you are trying to connect with. Example: `test.geysermc.org:19132`");
             return;
         }
 
         // Still allow people to not supply a port and fallback to 19132
         String[] fullAddress = args[0].split(":", 2);
         int port;
         if (fullAddress.length == 2) {
-            port = Integer.parseInt(fullAddress[1]);
+            try {
+                port = Integer.parseInt(fullAddress[1]);
+            } catch (NumberFormatException e) {
+                // can occur if e.g. "/geyser connectiontest <ip>:<port> is ran
+                sender.sendMessage("Not a valid port! Specify a valid numeric port.");
+                return;
+            }
         } else {
             port = 19132;
         }
+        String ip = fullAddress[0];
+
+        // Issue: people commonly checking placeholders
+        if (ip.equals("0.0.0.0") || ip.equals("<ip>")) {
+            sender.sendMessage(ip + " is not a valid IP, and instead a placeholder. Please specify the IP to check.");
+            return;
+        }
```

Comment:
This isn't really accurate for the first case - anyway, saying something along the lines of "enter the address you are trying to connect with" would probably be more useful

Reply:
I've split the cases 0.0.0.0 and the placeholder - i just want to avoid people changing "0.0.0.0" in the config thinking that that + ip have to match